client/src/components/TicketsSidebar/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import './style.css';
+
+const TicketsSidebar = ({ selected }) => (
+  <div className="ticket-sidebar">
+    <button className="ticket-sidebar__new-ticket">
+      <i className="fas fa-plus" />
+      {'          '}
+      Ticket
+    </button>
+    <div className="ticket-sidebar__links-container">
+      <Router>
+        <div className={`ticket-sidebar__links  ${selected === 'all' ? 'selected-tab' : ''}`}>
+          <Link to="/tickets/all-ticket" className={`ticket-sidebar__all-link ${selected === 'all' ? 'ticket-sidebar__all-link-selected' : ''}`}>
```

Comment:
The ```selected``` class seems to be a modifier that will change how the component looks/interacts but not the component itself. So we should use the modifier naming convention here.
```suggestion
          <Link to="/tickets/all-ticket" className={`ticket-sidebar__all-link ${selected === 'all' ? 'ticket-sidebar__all-link--selected' : ''}`}>
```